lib/sqlalchemy_softdelete/softdelete.py

```diff
@@ -11,9 +11,12 @@ def __init__(self, *args, **kwargs):
         self._super = super(SoftDeleteSession, self)
         kwargs['enable_baked_queries'] = False
         self._super.__init__(query_cls=SoftDeleteQuery, *args, **kwargs)
+        # Flag allowing you to disable the soft delete query.
+        self.disableSoftDeleteQuery = False
```

Comment:
Shouldn't this be snake case?  (I think your C# is showing :p )

Also, while making this a flag is fine, one python patter to avoid accidentally leaving the session with soft delete disabled would be to make this into a context manager.  E.g.:
```
from contextlib import contextmanager
...
@contextmanager
def disable_soft_delete_query(self):
    self._disable_soft_delete_query = True
    try:
        yield self
    finally:
        self._disable_soft_delete_query = False
```

Then usage looks like:
```
with sql_session.disable_soft_delete_query():
    sql_session.query(....)
```

Reply:
Good point on the snake case - updated.

Wish we could use the context manager change but I need the softdelete disabled on return from a view (and that's where the context manager would contain the scope of disable_soft_delete_query).